sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Utils.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.appcenter.storage;
+
+import com.google.gson.Gson;
+import com.microsoft.appcenter.http.HttpException;
+import com.microsoft.appcenter.http.HttpUtils;
+import com.microsoft.appcenter.utils.AppCenterLog;
+
+public final class Utils {
+    public static final Gson sGson = new Gson();
+
+    /**
+     * Handle API call failure.
```

Comment:
When a method is public, we will get a warning if we don't document parameter when using Javadoc syntax.